curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-algorithm-scripting/finders-keepers.english.md

```diff
@@ -9,7 +9,7 @@ forumTopicId: 16016
 
 ## Description
 <section id='description'>
-Create a function that looks through an array (first argument) and returns the first element in the array that passes a truth test (second argument). If no element passes the test, return undefined.
+Create a function that looks through an array, <code>arr</code>, and returns the first element in the array that passes a 'truth test'. That is to say, given an element <code>x</code>, the 'truth test' is passed if <code>func(x)</code> is <code>true</code>. If no element passes the test, return <code>undefined</code>.
```

Comment:
```suggestion
Create a function that looks through an array, `arr`, and returns the first element in the array that passes a 'truth test'. That is to say, given an element `x`, the 'truth test' is passed if `func(x)` is `true`. If no element passes the test, return `undefined`.
```

Reply:
@moT01 I think the comma after the word array and after the `arr` is not needed and maybe a slight rewording could make it a bit more concise.  What do you think?
```suggestion
Create a function that looks through an array `arr` and returns the first element in it that passes a 'truth test'. This means that given an element `x`, the 'truth test' is passed if `func(x)` is `true`. If no element passes the test, return `undefined`.
```

Reply:
I think those commas are needed, or maybe put it in parenthesis instead - something to split it up a little anyway. There's always room for improvement, but this was the change discussed in the issue, so I'm fine with it like this.